### PR TITLE
adds composer for paginatable collections

### PR DESCRIPTION
**Pull request by luizcarlos1405**
A simple composer that resolves quave:definitions paginated query.

### CODE REVIEW

File: resolvers.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+export const LIMIT = 7;
```
Comment (filipenevola) [inline resolvers.js]: Why export? And 10 would be better.

File: resolvers.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+
 export const resolvable = collection =>
```
Comment (filipenevola) [inline resolvers.js:5]: Maybe you could have a crudable that would compose resolvable and pagenable

File: resolvers.js [hunk 3]
```diff
@@ -17,6 +19,40 @@ export const resolvable = collection =>
     },
   });
 
+export const paginatable = collection =>
```
Comment (filipenevola) [inline resolvers.js]: I would guess that pagenable would be better but not sure.

**Review by filipenevola — CHANGES_REQUESTED** [review]

File: resolvers.js (hunk 3 above)
Comment (luizcarlos1405) [reply resolvers.js]: Oh agree, Webstorm didn't recognize `paginable` as a valid word so I thought it was wrong, but searching online it seems just as valid as `paginatable`.

**Pull request merged**